src/metrics-server/chart/templates/uds-package.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: metrics-server
+  namespace: {{ .Release.Namespace }}
+spec:
+  network:
+    allow:
+      - direction: Egress
+        remoteGenerated: KubeAPI
+        podLabels:
+          app.kubernetes.io/name: metrics-server
+      - direction: Ingress
+        podLabels:
+          app.kubernetes.io/name: metrics-server
+        remoteNamespaceLabels:
+          kubernetes.io/metadata.name: "istio-system"
+        port: 443
```

Comment:
Did you see an issue without this ingress policy? I removed it and everything seemed the same (passed validation, `kubectl top` worked, etc) - not following the need for this one 🤔

Reply:
could have been a bad/copy pasta--tbh dont remember now why, but happy to delete